[0m13:08:49.756531 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000016BD9047F50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000016BD8FD6590>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000016BD9024050>]}
[0m13:08:49.771531 [debug] [MainThread]: An error was encountered while trying to send an event


============================== 13:08:49.772528 | b46ad8d7-b56d-437b-b035-80461777dc60 ==============================
[0m13:08:49.772528 [info ] [MainThread]: Running with dbt=1.9.4
[0m13:08:49.786553 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\Fatoumatta\\.dbt', 'debug': 'False', 'fail_fast': 'False', 'log_path': 'logs', 'version_check': 'True', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'introspect': 'True', 'log_format': 'default', 'target_path': 'None', 'invocation_command': 'dbt init', 'send_anonymous_usage_stats': 'True'}
[0m13:08:59.492936 [debug] [MainThread]: Starter project path: C:\Users\Fatoumatta\AppData\Local\Programs\Python\Python311\Lib\site-packages\dbt\include\starter_project
[0m13:08:59.608849 [info ] [MainThread]: 
Your new dbt project "data_pipeline" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m13:08:59.610850 [info ] [MainThread]: Setting up your profile.
[0m13:12:06.121914 [info ] [MainThread]: Profile data_pipeline written to C:\Users\Fatoumatta\.dbt\profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m13:12:06.160587 [debug] [MainThread]: Command `dbt init` succeeded at 13:12:06.159588 after 197.25 seconds
[0m13:12:06.181900 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000016BD8DF6ED0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000016BDC932D90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000016BD8D17F50>]}
[0m13:12:06.183422 [debug] [MainThread]: An error was encountered while trying to send an event
[0m13:12:06.184436 [debug] [MainThread]: Flushing usage events
[0m14:30:57.141863 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000185433668D0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000018543343FD0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000018543305210>]}
[0m14:30:57.148860 [debug] [MainThread]: An error was encountered while trying to send an event


============================== 14:30:57.149861 | 983e0d3f-bb1a-44c9-adae-888e98826b66 ==============================
[0m14:30:57.149861 [info ] [MainThread]: Running with dbt=1.9.4
[0m14:30:57.152874 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'fail_fast': 'False', 'profiles_dir': 'C:\\Users\\Fatoumatta\\.dbt', 'log_path': 'logs', 'debug': 'False', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'log_format': 'default', 'static_parser': 'True', 'target_path': 'None', 'invocation_command': 'dbt init', 'send_anonymous_usage_stats': 'True'}
[0m14:31:14.109259 [debug] [MainThread]: Starter project path: C:\Users\Fatoumatta\AppData\Local\Programs\Python\Python311\Lib\site-packages\dbt\include\starter_project
[0m14:31:14.149809 [info ] [MainThread]: 
Your new dbt project "dbt_data_pipeline" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m14:31:14.153803 [info ] [MainThread]: Setting up your profile.
[0m14:34:23.914660 [info ] [MainThread]: Profile dbt_data_pipeline written to C:\Users\Fatoumatta\.dbt\profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m14:34:23.923841 [debug] [MainThread]: Command `dbt init` succeeded at 14:34:23.923841 after 206.91 seconds
[0m14:34:23.924829 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001854311EA10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001853CCC7A90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001853CCC7B10>]}
[0m14:34:23.931836 [debug] [MainThread]: An error was encountered while trying to send an event
[0m14:34:23.936866 [debug] [MainThread]: Flushing usage events
[0m21:54:57.347988 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002AC19949450>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002AC1994B210>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002AC1994ADD0>]}
[0m21:54:57.364106 [debug] [MainThread]: An error was encountered while trying to send an event


============================== 21:54:57.364106 | ccbd567c-7710-46e6-9efc-514e749ed5a2 ==============================
[0m21:54:57.364106 [info ] [MainThread]: Running with dbt=1.9.4
[0m21:54:57.367640 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'fail_fast': 'False', 'profiles_dir': 'C:\\Users\\Fatoumatta\\.dbt', 'log_path': 'logs', 'debug': 'False', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'False', 'quiet': 'False', 'no_print': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt compile', 'log_format': 'default', 'introspect': 'True', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m21:54:57.372189 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path C:\Users\Fatoumatta\Documents\Canada\Data Engineer\ELT pipeline\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m21:54:57.378215 [debug] [MainThread]: Command `dbt compile` failed at 21:54:57.378215 after 0.30 seconds
[0m21:54:57.380250 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002AC19942750>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002AC198D79D0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002AC198D5F50>]}
[0m21:54:57.380250 [debug] [MainThread]: An error was encountered while trying to send an event
[0m21:54:57.380250 [debug] [MainThread]: Flushing usage events
